ztp/argocd/resource-hooks/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+resources:
```

Comment:
please move this under ztp/gitops-subscriptions/argocd and better to sub directory as ztp/gitops-subscriptions/argocd/example and add test-sno.yaml there

Reply:
fixed